core/dataflow/src/main/resources/demo_map_udf.py

```diff
@@ -0,0 +1,3 @@
+def map_function(row, args):
```

Comment:
I saw a few files with name `demo_xxx`, are we planning to use and maintain them in a long term or it's just for demo purposes? If we plan to use them, then try to rename it and add some documentation.

Reply:
They are just for demo purposes so that other developers who wish to add UDFs can have some examples.